build.go

```diff
@@ -15,7 +15,7 @@ func build(k *Kong, ast interface{}) (app *Application, err error) {
 	v := reflect.ValueOf(ast)
 	iv := reflect.Indirect(v)
 	if v.Kind() != reflect.Ptr || iv.Kind() != reflect.Struct {
-		return nil, fmt.Errorf("expected a pointer to a struct but got %T", ast)
+		return nil, Errors().ExpectedPointerToStructButGot(ast)
```

Comment:
I really do not like this API. I would be happy with an API something like the following, where `Lang` is of a type that will look up the strings in a translation table. I don't know if such an API already exists.

```
return nil, Lang.Errorf("expected a pointer to a struct but got %[1]T", ast)
```